### PR TITLE
Create C8300-6T

### PR DESCRIPTION
**Pull request by davidabuckner**
I forgot to update the slug to the correct model.

### CODE REVIEW

File: device-types/Cisco/C8300-6T [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: Cisco
+model: Catalyst 8300-6T
+slug: c8300-12x
+part_number: C8300-6T
```
Comment (DanSheps) [inline device-types/Cisco/C8300-6T:5]: This is not correct.  There is a more appopriate model number, one of:

C8300-1N1S-4T2X
C8300-1N1S-6T
C8300-2N2S-6T
C8300-2N2S-4T2X

File: device-types/Cisco/C8300-6T [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    type: usb-micro-b
+  - name: usbc
+    type: usb-c
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 650
```
Comment (DanSheps) [inline device-types/Cisco/C8300-6T:21]: Power supplies are modular, these should be setup as modules with each power supply as a module-type:

PWR-CC1-250WAC
PWR-CC1-500WAC
PWR-CC1-400WDC
PWR-CC1-400WHV

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @davidabuckner

**Comment by danner26** [comment]
If you still want to contribute we would be happy to review it again. The original maintainers are busy but I am here to make sure the PRs get worked on
If you would like to contribute please feel free to reopen this PR, update your branch against our master, and then fix any issues. From there I will review it

**Pull request closed without merging**